src/server/utils.js

```diff
@@ -1,22 +1,36 @@
 const { PrismaClient } = require('@prisma/client');
 
-const { SERVER_STATUS_CODE, SERVER_ERROR_MESSAGE } = require('./config');
-
 const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
 
-const { SERVER_ERROR_MESSAGE, SERVER_STATUS_CODE } = require('./config');
+const { SERVER_ERROR_MESSAGE } = require('./config');
 
 const prisma = new PrismaClient();
 
+const saltRounds = 10;
+
 const capitalizeFirstLetter = (string) =>
     string.replace(/\b\w/g, (c) => c.toUpperCase());
 
+const hashedPassword = (password) => bcrypt.hashSync(password, saltRounds);
+
+const createToken = (payload) => jwt.sign(payload, secret);
+
+const checkPassword = async (textPassword, hashedPassword) => {
+    try {
+        return await bcrypt.compare(textPassword, hashedPassword);
```

Comment:
nice use of bcrypt and try and catch great job!

Reply:
overall great job on the backend proud of you guys!!

Reply:
I love your use of console.log.